vendor/vendor.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This is not necessarily this PR's fault but strange that this project never had a vendor.json. Now you have added one but its contents only contain one dependency and not the others that this actually uses. Would suggest removing this file for now and we should file an issue and create a correct vendor.json with all dependencies.